.github/workflows/ci.yml

```
@@ -246,6 +263,19 @@ jobs:
           # Check with all features.
           cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p wgpu-hal --all-features
           cargo clippy --target ${{ matrix.target }} ${{ matrix.extra-flags }} -p wgpu --all-features
+        
+      # Building for no_std platforms where every feature is enabled except "std".
+      - name: check no_std
+        if: matrix.kind == 'no_std'
```

```suggestion
        if: matrix.kind == 'no-std'
```

Used different strings for no-std

---

Ah silly mistake on my part, thanks for fixing that!